src/Symfony/Bundle/FrameworkBundle/Resources/config/schema/symfony-1.0.xsd

```diff
@@ -745,6 +747,35 @@
         </xsd:choice>
     </xsd:complexType>
 
+    <xsd:complexType name="http_client_caching">
+        <xsd:sequence maxOccurs="unbounded">
+            <xsd:element name="cacheable-status-code" minOccurs="0" maxOccurs="unbounded">
+                <xsd:simpleType>
+                    <xsd:list itemType="xsd:integer"/>
+                </xsd:simpleType>
+            </xsd:element>
+            <xsd:element name="conditionally-cacheable-status-code" minOccurs="0" maxOccurs="unbounded">
+                <xsd:simpleType>
+                    <xsd:list itemType="xsd:integer"/>
+                </xsd:simpleType>
+            </xsd:element>
+            <xsd:element name="cacheable-method" minOccurs="0" maxOccurs="unbounded">
+                <xsd:simpleType>
+                    <xsd:list itemType="xsd:string"/>
```

Comment:
Why an `xsd:list` ? This does not make sense to me. The `cacheable-method` method element should *be* as `xsd:string` directly (see the `transition` type for the workflow configuration line 478 of this file for an example)